src/I18nService.js

```diff
@@ -26,14 +26,33 @@ export class I18nService extends EventEmitter {
     }
   }
 
-  t(label, args) {
-    return this.translate(label, this.language, args)
+  t(label, args, mark) {
+    return this.translate(label, this.language, args, mark)
   }
 
-  translate(label, language, args) {
+  translate(label, language, args, mark) {
     this.ensureSupportedLanguage(language)
 
     const dictionary = this.getDictionary(language)
+    const marked = args?.[mark]
+
+    if (mark !== undefined && !isNaN(marked)) {
```

Comment:
```suggestion
    if (mark !== undefined && !isNaN(args?.[mark])) {
      const marked = args?.[mark]
```

Reply:
The `?.` operator tests if the right side is defined AND if the left side exists inside the right side.